code/game/objects/structures/watercloset.dm

```diff
@@ -405,6 +405,7 @@
 	if(has_water_reclaimer && reagents.total_volume < reagents.maximum_volume)
 		reagents.add_reagent(dispensedreagent, 1)
```

Comment:
With changes to processing, we're now passing a delta_time through to process() procs

Below is a quick and dirty example of the kind of change requested.
```
/obj/structure/sink/process(delta_time)
```
```
// Dispense 1 reagent every 2 seconds.
reagents.add_reagent(dispensedreagent, delta_time * 0.5)
```

This way if the fluid subsystem skips processing ticks for any reason, the sink still fills up appropriately.